pvlib/irradiance.py

```diff
@@ -613,10 +613,23 @@ def isotropic(surface_tilt, dhi):
     dhi : numeric
         Diffuse horizontal irradiance. [Wm⁻²] DHI must be >=0.
 
+    return_components : bool, default False
+        Flag used to decide whether to return the calculated diffuse components
+        or not. If `False`, ``sky_diffuse`` is returned. If `True`,
```

Comment:
```suggestion
        or not. If `False`, ``poa_sky_diffuse`` is returned. If `True`,
```
Assuming this is in reference to the names in the `Returns` section